examples/hybrid/define_post_processing.jl

```diff
@@ -126,13 +126,13 @@ end
 
 # Dispatcher:
 paperplots_baro_wave(args...) =
-    paperplots_baro_wave(Val(energy_name()), Val(moisture_mode()), args...)
+    paperplots_baro_wave(energy_form(), moisture_model(), args...)
 
-paperplots_baro_wave(::Val{:ρθ}, ::Val{:dry}, args...) =
+paperplots_baro_wave(::ρθForm, ::DryModel, args...) =
     paperplots_baro_wave_ρθ(args...)
-paperplots_baro_wave(::Val{:ρe}, ::Val{:dry}, args...) =
+paperplots_baro_wave(::ρeForm, ::DryModel, args...) =
     paperplots_baro_wave_ρe(args...)
-paperplots_baro_wave(::Val{:ρe}, ::Val{:equil}, args...) =
+paperplots_baro_wave(::ρeForm, ::EquilModel, args...) =
```

Comment:
Can we call them `PotentialTemperature`, `TotalEnergy` instead of `ρθForm`, `ρeForm`

Reply:
And `EquilMoistModel`

Reply:
And maybe `EnergyVariable` instead of `EnergyForm`?

Reply:
Sure, no problem. I kind of like the `ρθForm` and `ρeForm`, just because `PotentialTemperature` could cause confusion with `θ` itself. But I don't have strong opinions on names, here.

Reply:
I went with @jiahe23's initial suggestion+ `InternalEnergy`